docs/application/native/guides/text-input/capi-ui-gesture.md

```diff
@@ -0,0 +1,247 @@
+# Gesture
+
+`Gesture` is a movement or position of the hand, body, head, and face, and it expresses emotion, opinion, and so on.
+
+As there are many gesture types, so there are various methods to recognize gestures on the device.
+For example, hand gestures are recognized, based on gyro sensor data. And, head movements can be detected with a visual camera and video frames.
+According to the targeted gesture, the device and the input sensor, the recognition method can be changed.
+
+In the current Gesture, only wearable device is targeted and only hand gesture is recognized from input sensor data.
+
+The main features of the Gesture API include:
+
+- Getting the supported gesture type
+
+  You can [get the supported gesture type](#info_gesture) to be recognized.
+
+- Setting gesture options
+
+  You can [set options](#set_option) such as a hand gesture type and option.
+
+- Preparing the Gesture service for use
+
+  You can [connect the background Gesture daemon](#prepare) to be able to operate the gesture.
+
+
+<a name="basic_gesture"></a>
+## Basic Gesture processes
+
+There are processes you should follow in order to get the recognized gesture. Using Gesture, you can:
```

Comment:
Suggest rephrasing to:

To get the recognized gesture, you must follow the following processes:

Please verify for technical accuracy

Reply:
Thank you for your recommendation. It looks good to express the intention.
I changed this sentence. Please review this again.